### ISSUE TITLE
Typeform iOS Viewer

### ISSUE THREAD

**Issue by joetischler** [r3trosteve/typeform-ios#1]
Would love to see some sort of similar build with actually filling out forms.  I'm new(er) to typeform and am wondering if it can be used easily within an iOS app deliver surveys to users.

**Comment by r3trosteve** [comment]
I just serve the typeform that I generate using typeform.io (their new forms api) into a UIWebView and it works quite well, by all accounts... what else would you like to see?

**Comment by joetischler** [comment]
Is that part built into this repo?  I saw a transition for UIWebView but all it shows is the edit button.  (Doing some research on survey tools, kind of an iOS noob as well 😊)

Right now we are using ResearchKit and we're not too fond with it ATM - no easy way to edit forms on the fly and it's not cross platform

**Comment by r3trosteve** [comment]
Yes, once you create the form you can view it in the web view in this app.

Are you building health assessment / health research / health apps? I'd be interested to understand more. I'm CTO of Nuwe, a developer platform for building health apps.

**Comment by joetischler** [comment]
We are indeed.  I'm talking a look at your platform now - very cool!  I'll take it to my team and see what they think.

For some reason I'm not able to view built out forms in the simulator for this current version.  May be something on my side?  Will dive in some more, we're really hoping to get cranking so we can replace RK.
